test/test_verify.py

```diff
@@ -464,7 +480,10 @@ def runTestSet(driver, testSet, nameSalt, pressure):
         testSnowpipeStreamingStringJson, testSnowpipeStreamingStringAvro,
         testMultipleTopicToOneTableSnowpipeStreaming, testMultipleTopicToOneTableSnowpipe,
         testSchemaMapping,
-        testAutoTableCreation, testAutoTableCreationTopic2Table
+        testAutoTableCreation, testAutoTableCreationTopic2Table,
+        testSchemaEvolutionJson, testSchemaEvolutionAvroSR,
+        testSchemaEvolutionWithAutoTableCreationJson, testSchemaEvolutionWithAutoTableCreationAvroSR,
+        testSchemaEvolutionNonNullableJson
```

Comment:
this is not scalable in future. 
Can you may be create a jira to refactor this into a map of test name and whether it is enabled in confluent or kafka? (Someone will pick it up)